Library/Formula/jetty.rb

```diff
@@ -1,16 +1,19 @@
 class Jetty < Formula
   homepage "http://www.eclipse.org/jetty/"
-  url "http://download.eclipse.org/jetty/9.2.10.v20150310/dist/jetty-distribution-9.2.10.v20150310.tar.gz"
-  version "9.2.10.v20150310"
-  sha256 "0034587a3dcfdab23fca28291e5e22ae7ff6e80f9f4c7a67f5b2b7777a84b40e"
+  desc "A Servlet Container and Http Server in Java"
```

Comment:
It's pending the merge of another PR, but after that it should be relatively imminent.